OntologyMapper/src/OntologyMappingManager.cs

```diff
@@ -51,9 +52,9 @@ public bool TryGetPropertyProjection(string outputDtmiFilter, string outputPrope
             return false;
         }
 
-        public bool TryGetFillProperty(string outputDtmiFilter, string outputPropertyName, out FillProperty? fillProperty)
+        public bool TryGetFillProperty(string outputDtmi, string outputPropertyName, out FillProperty? fillProperty)
         {
-            fillProperty = OntologyMapping.FillProperties.FirstOrDefault(e => e.OutputDtmiFilter == outputDtmiFilter && e.OutputPropertyName == outputPropertyName);
+            fillProperty = OntologyMapping.FillProperties.FirstOrDefault(e => (e.OutputDtmiFilter == "*" || Regex.Match(outputDtmi, e.OutputDtmiFilter, RegexOptions.IgnoreCase).Success) && e.OutputPropertyName == outputPropertyName);
```

Comment:
Isn't the star already defined as one of the filters?

Also, is the fillProperties array ordered? Should "``\w*Space\w*" be above star? because both satisfy the constraint.